corehq/apps/app_manager/suite_xml/sections/resources.py

```diff
@@ -31,7 +31,7 @@ def get_section_elements(self):
                 local=path,
                 remote=remote_path,
             )
-            if self.app.build_version >= '2.9':
+            if self.app.build_version >= b'2.9':
```

Comment:
Similar to the above - this feels like the wrong approach.  I just tested it out, and this breaks in the python3 version of distutils.  I recommend changing these lines to:
```python
if self.app.build_version >= LooseVersion('2.9')
```